homeassistant/components/nanoleaf/light.py

```diff
@@ -53,23 +55,32 @@
 )
 
 
+def get_serial_number(host: str) -> str:
+    return requests.get(f"http://{host}/device_info").json()["serialNumber"]
```

Comment:
This is part of the API that needs to be abstracted into the `pynanoleaf` package.

Reply:
Ok, I've created Oro/pynanoleaf#1